.travis.yml

```diff
@@ -7,8 +13,30 @@ notifications:
     email: false
 addons:
   apt:
+    sources:
+    - ubuntu-toolchain-r-test
     packages:
     - libgmp-dev
+    - libmpfr-dev
+    - libmpc-dev
+    - binutils-dev
+    - g++-4.7
+    - g++-5
+    - gcc-5
+    
+install:
+  - export JULIA_SOURCE_DIR=`pwd`
+  - export TEST_CPP="no"
+
+  - git clone https://github.com/symengine/symengine symengine-cpp
+  - cd symengine-cpp
+  - git checkout `cat ../symengine_version.txt`
```

Comment:
This statement won't work because there isn't any `symengine_version.txt` in the SymEngine.jl repository symengine.rb.